### PR TITLE
Happ 259

### PR DESCRIPTION
**Pull request by AmirShayegh**

### CODE REVIEW

File: BCVaccineCard/BCVaccineCard.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		2908647A27A1B08E0003F9E3 /* LocalAuthManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2908647927A1B08E0003F9E3 /* LocalAuthManager.swift */; };
```
Comment (ConnorOgilvie91) [inline BCVaccineCard/BCVaccineCard.xcodeproj/project.pbxproj:10]: Can you make sure that all files added in this PR are added to both targets?

Comment (AmirShayegh) [reply BCVaccineCard/BCVaccineCard.xcodeproj/project.pbxproj:10]: yup

File: BCVaccineCard/BCVaccineCard/AppDelegate.swift [hunk 2]
```diff
@@ -136,3 +150,49 @@ extension AppDelegate {
     }
 }
 
+
+// MARK: Local Auth
+extension AppDelegate {
+//    private func performLocalAuth() {
```
Comment (ConnorOgilvie91) [inline BCVaccineCard/BCVaccineCard/AppDelegate.swift:156]: Do we need this anymore?

File: BCVaccineCard/BCVaccineCard/Utility/Constants/Strings/Strings.swift [hunk 3]
```diff
@@ -232,6 +232,21 @@ extension String {
...
+    
+    // MARK: Local Auth
+    static var localAuthPrivacyText: String { return "LocalAuthPrivacyText".localized }
+    static var privacy: String { return "Privacy".localized }
+    static var protectYourPersonalInformation: String { return "ProtectYourPersonalInformation".localized }
+    static var localAuthViewDescription: String { return "LocalAuthViewDescription".localized }
+    static var learnMoreAboutLocalAuth: String { return "LearnMoreAboutLocalAuth".localized }
+    
+    static var useTouchId: String { return "UseTouchId".localized }
+    static var usePassCode: String { return "usePassCode".localized }
```
Comment (ConnorOgilvie91) [inline BCVaccineCard/BCVaccineCard/Utility/Constants/Strings/Strings.swift:244]: Just for my OCD to keep things consistent, can you upper case the key to "UsePassCode"

File: BCVaccineCard/BCVaccineCard/Utility/Constants/Strings/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -189,3 +189,16 @@
...
+
+"LocalAuthPrivacyText" = "Biometric authenication is a security feature to protect your personal health information and are required to log on My Health BC.\n\nYour health information and the health information of your spouse, dependents, and claim details will be stored in your mobile device. Should you elect to use Touch ID offered by Apple Inc., or fingerprint sign in functionality offered by Android, or any other type of biometric identification managed on your device (collectively “biometrics”), you agree with acknowledge and consent to the terms and conditions, which can be accessed at here.";
+"Privacy" = "Privacy";
+"LearnMoreAboutLocalAuth" = "Learn more about enabling biometric on your device";
+"ProtectYourPersonalInformation" = "Protect your personal information";
+"LocalAuthViewDescription" = "Unlock My Health BC with biometric authentication is mandatory, to make sure your health information safe and secure.";
+"LearnMoreAboutLocalAuth" = "Learn more about enabling biometric on your device";
+"UseTouchId" = "Use Touch ID";
+"usePassCode" = "Use passcode";
```
Comment (ConnorOgilvie91) [inline BCVaccineCard/BCVaccineCard/Utility/Constants/Strings/en.lproj/Localizable.strings:200]: and here - "UsePassCode"

**Pull request merged**